data/electronToChromium.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  "1.5": 54,
```

Comment:
@kevinsawicki let me know how we can help to get atom to use this (if you're interested)